### PR TITLE
Adding e2e tests from Appsody

### LINKED ISSUE #5: Add e2e tests
Add e2e tests for the features the Application Runtimes Operator supports (route, scaling/HPA, knative, persistence, etc).

### PR DESCRIPTION
**Pull request by JusteenR**
**What this PR does / why we need it?**:

- Includes e2e tests from Appsody that have been changed for the Runtime Operator

**Which issue(s) this PR fixes**:

Fixes #5

### CODE REVIEW

**Comment by leochr** [comment]
@JusteenR Thanks for the PR. It'll be good to avoid referencing or including modules from Open Liberty and Appsody operator as Application Runtime operator will be the base for those operators. If any modules are needed from them, please include them as part of this operator. Thanks.

**Comment by JusteenR** [comment]
@leochr no modules from Open Liberty or Appsody were referenced, I copied them over directly to this operator and changed the naming for the applicable tests

File: go.mod [hunk 1]
```diff
@@ -3,11 +3,10 @@ module github.com/application-runtimes/operator
 go 1.13
 
 require (
-	contrib.go.opencensus.io/exporter/ocagent v0.4.9 // indirect
 	github.com/Azure/go-autorest v13.0.2+incompatible // indirect
+	github.com/OpenLiberty/open-liberty-operator v0.3.0
```
Comment (leochr) [inline go.mod]: Open Liberty is referenced here.

**Comment by leochr** [comment]
@JusteenR The tests don't reference Open Liberty or Appsody. But the go related files do, which is causing the OL/Appsody modules to be included under `vendor` directory.

**Pull request merged**